src/rippled/lib/txSummary/index.js

```diff
@@ -39,7 +37,7 @@ const getInstructions = (tx, meta) => {
   return mappingFn ? mappingFn(tx, meta) : {}
 }
 
-const summarizeTransaction = (d, details = false) => ({
+export const summarizeTransaction = (d, details = false) => ({
```

Comment:
This is already exported below, on line 55 - it doesn't also need to be exported here.

Reply:
I was having issues building until I changed this.  I don't know what suddenly caused it.